### PR TITLE
[fastlane] Add new team_id option to latest_testflight_build_number

### PR DESCRIPTION
**Pull request by KrauseFx**
Fixes https://github.com/fastlane/fastlane/issues/1464

### CODE REVIEW

File: fastlane/lib/fastlane/actions/latest_testflight_build_number.rb [hunk 1]
```diff
@@ -75,8 +76,12 @@ def self.available_options
                                        env_name: "INTITIAL_BUILD_NUMBER",
                                        description: "sets the build number to given value if no build is in current train",
                                        optional: true,
-                                       is_string: false)
-
+                                       is_string: false),
+          FastlaneCore::ConfigItem.new(key: :team_id,
+                                       env_name: "FASTLANE_TEAM_ID",
+                                       description: "optional: Your team ID",
```
Comment (asfalcone) [inline fastlane/lib/fastlane/actions/latest_testflight_build_number.rb]: Do we need "optional" in the description? We don't do this for other ConfigItems.

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/latest_testflight_build_number.rb]: Great point 👍

**Comment by KrauseFx** [comment]
Updated 👍

**Comment by asfalcone** [comment]
Great thanks 🚀

**Comment by jlubeck** [comment]
https://github.com/fastlane/fastlane/issues/5366 this is should also be marked as fixed with this commit, BUT, after installing 1.100.0, fastlane just hangs on the latest_testflight_build_number step.

I tried running with verbose, but nothing shows up.

**Comment by KrauseFx** [comment]
@jlubeck Nice catch, I'll push another PR in a minute

**Comment by KrauseFx** [comment]
@jlubeck Pushed https://github.com/fastlane/fastlane/pull/5725 👍

**Pull request merged**